packages/hash/api/src/model/ontology/property-type.model.ts

```diff
@@ -62,7 +63,11 @@ export default class {
     return persistedPropertyTypes.map(
       (persistedPropertyType) =>
         new PropertyTypeModel({
-          schema: persistedPropertyType.inner,
+          /**
+           * @todo and a warning, these type casts are here to satisfy the type system package.
```

Comment:
I'm not certain about this message. I think it might be giving the wrong idea. Is this really to satisfy the type system package? Or is this because our generated OpenAPI client isn't good enough?

Reply:
I know it's quite verbose/long but changed it in f5fb24ad45ccbe41425b7b56c9271eb7a7ce25df. I want to have these warnings as a blaring piece until we sort it out.

I think the shape is actually matching, but we're not asserting stuff like the `$id`s and `$refs` matching the template literal for instance.

Reply:
Much better, thank you